sdk/go/common/apitype/templates.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016-2025, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apitype
+
+// A pulumi template remote where the source URL contains
+// a valid Pulumi.yaml file.
+type PulumiTemplateRemote struct {
+	ProjectTemplate
+	Name        string `json:"name"`       // The name of the template
+	SourceName  string `json:"sourceName"` // The name of the template source - for display purposes
+	TemplateURL string `json:"sourceURL"`  // The unique url that identifies the template to the service
+	Runtime     string `json:"runtime"`    // The runtime of the template
+}
+
+type GetOrgTemplatesResponse struct {
+	Templates        map[string][]*PulumiTemplateRemote `json:"templates"`
+	OrgHasTemplates  bool                               `json:"orgHasTemplates"`
+	HasAccessError   bool                               `json:"hasAccessError"`
+	HasUpstreamError bool                               `json:"hasUpstreamError"`
```

Comment:
This could maybe use a docstring to explain when exactly we expect these kinds of errors.